incubator/crate/templates/statefulset.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: "apps/v1beta1"
```

Comment:
Is there any reason you chose `apps/v1beta1` for the statefulset? Statefulset has been in `apps/v1` since k8s 1.9.

Reply:
Oh thanks for pointing that out. No reasons other than historical ones, it's an old chart :p